django/db/backends/sqlite3/operations.py

```diff
@@ -107,6 +107,19 @@ def _quote_params_for_last_executed_query(self, params):
         """
         Only for last_executed_query! Don't use this to execute SQL queries!
         """
+        # This function is limited both by SQLITE_LIMIT_VARIABLE_NUMBER (the
+        # number of paramters, default = 999) and SQLITE_MAX_COLUMN (the
+        # number of return values, default = 2000). Since Python's sqlite3
+        # module doesn't expose the get_limit() C API, assume the default
+        # limits are in effect and split the work in batches if needed.
+        BATCH_SIZE = 999
+        if len(params) > BATCH_SIZE:
+            results = []
+            for index in range(0, len(params), BATCH_SIZE):
+                chunk = params[index:index + BATCH_SIZE]
+                results += self._quote_params_for_last_executed_query(chunk)
```

Comment:
When the function is called for a single batch, it returns `fetchone()`'s return value; on my system (Python 2.7), that's a tuple. Defining `results` above as `()` should fix the resulting type error (but that does not explain https://code.djangoproject.com/ticket/26063#comment:9)